apps/src/templates/progress/progressHelpers.js

```diff
@@ -156,6 +156,17 @@ export function lessonHasLevels(lesson) {
   return !!lesson.levels?.length;
 }
 
+export const feedbackLeft = progress =>
+  progress?.teacherFeedbackReviewState !== 'keepWorking' &&
+  progress?.teacherFeedbackNew;
+
+export const studentNeedsFeedback = (progress, level) =>
+  progress &&
+  progress.status !== LevelStatus.not_tried &&
```

Comment:
Does a student need feedback if the level is in progress but not submitted?

Reply:
I thought that's what we landed on in the last office hours. I'm happy to revisit that during the bug bash or whatever next opportunity we have.